backend/tracim_backend/lib/proxy/proxy.py

```diff
@@ -6,6 +6,8 @@
 import requests
 from pyramid.response import Response as PyramidResponse
 from requests import Response as RequestsResponse
+from requests.auth import HTTPBasicAuth, AuthBase
+
 from tracim_backend.lib.utils.request import TracimRequest
 
 HOP_BY_HOP_HEADER_HTTP = (
```

Comment:
note: on line 23, this is not a `FIXME` but a `HACK` comment -we have no reason for the moment to remove this hack

Reply:
done.